ocaml/typing/printtyped.ml

```diff
@@ -378,6 +378,11 @@ and expression_extra i ppf x attrs =
   | Texp_newtype (s, lay) ->
       line i ppf "Texp_newtype %a\n" (typevar_jkind ~print_quote:false) (s, lay);
       attributes i ppf attrs;
+  | Texp_mode_coerce modes ->
+      let modes = (modes :> string Location.loc list Location.loc) in
+      line i ppf "Texp_mode_coerce \"%s\"\n"
```

Comment:
Nit: do we need the quotes around modes? I think the types in `Texp_coerce` are not wrapped in quotes. I think this file mostly uses newlines and indentation for disambiguation.

Reply:
I think there are plenty of instances where this file uses quotes, usually to disambiguate whether something is an identifier (quotes) vs. a Typedtree field/constructor name (no quotes). Modes are identifiers.

It's probably a bit better to wrap each mode in quotes, I've done that now.